phpBB/adm/style/acp_main.html

```diff
@@ -94,15 +94,45 @@
 
 	<!-- EVENT acp_main_notice_after -->
 
+	<table class="table1 no-header" style="width: auto;" data-no-responsive-header="true">
+		<caption>{L_SOFTWARE_HEADING}</caption>
+		<col class="col1" /><col class="col2" />
+	<thead>
+	<tr>
+		<th>{L_SOFTWARE_PRODUCT}</th>
+		<th>{L_SOFTWARE_VERSION}</th>
+	</tr>
+	</thead>
+	<tbody>
+	<tr>
+		<td>{L_BOARD_VERSION}</td>
+	<!-- IF S_VERSIONCHECK -->
+		<td><strong><a href="{U_VERSIONCHECK}" <!-- IF S_VERSION_UP_TO_DATE -->style="color: #228822;" <!-- ELSEIF not S_VERSIONCHECK_FAIL -->style="color: #BC2A4D;" <!-- ENDIF -->title="{L_MORE_INFORMATION}">{BOARD_VERSION}</a></strong> [&nbsp;<a href="{U_VERSIONCHECK_FORCE}">{L_VERSIONCHECK_FORCE_UPDATE}</a>&nbsp;]
+		</td>
+	<!-- ELSE -->
+		<td><strong>{BOARD_VERSION}</strong></td>
+	<!-- ENDIF -->
+	</tr>
+	<tr>
+		<td>{L_PHP_VERSION}</td>
+		<td><strong>{PHP_VERSION_INFO}</strong></td>
+	</tr>
+	<tr>
+		<td>{L_DATABASE_SERVER_INFO}</td>
+		<td><strong>{DATABASE_INFO}</strong></td>
+	</tr>
+	</tbody>
+	</table>
+
 	<table class="table1 two-columns no-header" data-no-responsive-header="true">
 		<caption>{L_FORUM_STATS}</caption>
 		<col class="col1" /><col class="col2" /><col class="col1" /><col class="col2" />
 	<thead>
 	<tr>
-		<th>{L_STATISTIC}</th>
-		<th>{L_VALUE}</th>
-		<th>{L_STATISTIC}</th>
-		<th>{L_VALUE}</th>
+		<th style="width: 25%">{L_STATISTIC}</th>
```

Comment:
you are right, it's not really consistent and the design of this part of the ACP is a bit unique in the ACP.

I think we can merge it like that.